tests/gk_code/test_gk_output_reader_gs2.py

```diff
@@ -83,10 +84,40 @@ def test_infer_path_from_input_file_gs2():
     assert output_path == Path("dir/to/input_file.out.nc")
 
 
+def test_gs2_read_omega_file(tmp_path):
+    """Can we match growth rate/frequency from netCDF file"""
+
+    path = template_dir / "outputs" / "GS2_linear"
+    pyro = Pyro(gk_file=path / "gs2.in", name="test_gk_output_gs2")
+    pyro.load_gk_output()
+
+    netcdf_data = nc.Dataset(template_dir / "outputs" / "GS2_linear" / "gs2.out.nc")
+
+    cdf_mode_freq = netcdf_data["omega"][-1, 0, 0, 0]
+    cdf_gamma = netcdf_data["omega"][-1, 0, 0, 1]
```

Comment:
Best practice is to use `with nc.Dataset`:

```suggestion
    with nc.Dataset(path / "gs2.out.nc") as netcdf_data:
        cdf_mode_freq = netcdf_data["omega"][-1, 0, 0, 0]
        cdf_gamma = netcdf_data["omega"][-1, 0, 0, 1]
```

(also reuses `path`)